.circleci/bazelrc

```diff
@@ -4,6 +4,10 @@ build --announce_rc
 build --copt -O0
 build --disk_cache=/tmp/bazel-disk-cache
 
+# Stratum Bazel remote cache.
+# TODO: update url and figure out user/pw loading from env vars.
+build --remote_cache=http://user:pass@143.198.55.252:8080
```

Comment:
fix

Reply:
```
      BAZEL_CACHE_USER: github
      BAZEL_CACHE_PWD: ${{ secrets.BAZEL_CACHE_PWD }}
      BAZEL_CACHE_URL: pins-bazel-cache.onf.dev:9090
```

```        mkdir -p /tmp/repo-cache
        BAZEL_OPTS="--repository_cache=/tmp/repo-cache"
        [ ! -z "$BAZEL_CACHE_USER" ] && [ ! -z "$BAZEL_CACHE_PWD" ] && \
          AUTH="${BAZEL_CACHE_USER}:${BAZEL_CACHE_PWD}@"
        BAZEL_OPTS+=" --remote_cache=https://${AUTH}${BAZEL_CACHE_URL}"
        BAZEL_OPTS+=" --remote_download_minimal"
        bazel build ${BAZEL_OPTS} //...
        bazel test ${BAZEL_OPTS} //...
```

Reply:
Since bazelrc files do not support env var interpolation, we probably can use this workaround: https://github.com/bazelbuild/bazel/issues/10904#issuecomment-655026832